CMakeLists.txt

```diff
@@ -114,7 +115,7 @@ set(GTK_MINIMUM             3.4.0)
 set(GLIB_MINIMUM            2.50.1)
 set(GIO_MINIMUM             2.26.0)
 set(LIBAPPINDICATOR_MINIMUM 0.4.90)
-set(QT5_MINIMUM             5.2)
+set(QT_MINIMUM              5.6)
```

Comment:
Is this bump necessary?

Not a blocker; just asking so that I'll know why it was needed

Reply:
In line with other recent version bumps (C++17, Glib 2.50.1), Qt 5.6 was released in 2016 which seems acceptable; it is also the first LTS version in 5.x series which is an improvement over 5.2. The actual decision was taken when it came to c7004a839d09704ec6b63a9e466bebfa7fcf422c which required adding new `#if` block since the deprecated declaration was removed in 6.0; while deprecation warnings are still issued (when building against 6.0+), this particular place wouldn't require a check in the future.

Although being concervative here, I'm actually okay with bumping it even further, e.g. to 5.9 (LTS) released in 2017, given that C++17 wasn't fully implemented or considered stable enough in compilers released in 2017 anyway. Didn't check DistroWatch for 5.9 availability though.

Reply:
AFAIK, For Deb-based: 5.9.5 in Ubuntu 18.04 (oldest LTS) and 5.11.3 in Debian (oldstable)